content/en/tracing/profiler/getting_started.md

```diff
@@ -66,7 +66,7 @@ The Datadog Profiler requires [JDK Flight Recorder][1]. The Datadog Profiler lib
 | `DD_VERSION`                                     | String        | The version of your service.                             |
 | `DD_TAGS`                                        | String        | Tags to apply to an uploaded profile. Must be a list of `<key>:<value>` separated by commas such as: `layer:api, team:intake`.  |
 
-- By default, The Datadog Profiler keeps the bottom 256 frames in a Java profile. If your application creates stacks deeper than 256 frames, you can increase the threshold by passing a `-XX:FlightRecorderOptions=stackdepth=` option to your JVM.  For example:
+- If your application creates stacks deeper than 256 frames, you can increase the threshold by customizing the `-XX:FlightRecorderOptions=stackdepth=` option. If you do not specify a `stackdepth` the JVM default is 64.
```

Comment:
> If you do not specify a `stackdepth` the JVM default is 64.
But the default experience here seems to be 256 depth if you copy & paste. Should we put this in our troubleshooting docs and when we see 'truncated' frames, send them to the troubelshooting section to increase stackdepth?

Reply:
That's a good point @GaurabAryal to leave this page as concise as possible

Reply:
@GaurabAryal @hugokash removed that section